doc/reference_en.html

```diff
@@ -5740,6 +5740,10 @@ <h3 class="not-for-contents">1. Why not to use systems like map-reduce?</h3>
 ===isNaN(x)===
 Accepts Float32 and Float64 and returns UInt8 equal to 1 if the argument is a NaN, otherwise 0.
 
+===hasColumnInTable('database', 'table', 'column')===
```

Comment:
Not documented, that function throws exception if table does not exist.
Not documented, that for nested data type, it checks for existence of elements and not works for name of nested itself.